allensdk/brain_observatory/behavior/behavior_metadata.py

```diff
@@ -139,3 +147,220 @@ def get_task_parameters(data: Dict) -> Dict:
     task_parameters['n_stimulus_frames'] = n_stimulus_frames
 
     return task_parameters
+
+
+class BehaviorMetadata:
+    """Container class for behavior metadata"""
+    def __init__(self, extractor: Any,
+                 stimulus_timestamps: np.ndarray,
+                 behavior_stimulus_file: pd.DataFrame):
```

Comment:
Assuming this is the same thing that is returned by `BehaviorOphysSession.api._behavior_stimulus_file()`, in my experience it is just a `dict`, not a `DataFrame`.

Reply:
@danielsf yes you are right. `BehaviorDataTransforms. _behavior_stimulus_file() incorrectly had the return type as dataframe. I updated both types.